packages/admin-stories/src/docs/form/Overview.stories.mdx

```diff
@@ -1,15 +1,295 @@
+  `}
+/>
+
+<Source
+    language="ts"
+    code={dedent`
+        // employees/EmployeeForm.gql.ts
+        export const createEmployeeMutation = gql\`
+            mutation CreateEmployee($input: CreateEmployeeInput!) {
+                createEmployee(input: $input) {
+                    id
+                    name
+                }
+            }
+        \`;
+  `}
+/>
+
+If the form is short and simple, you can also define the mutation at the top of the same file.
+
+<Source
+    language="tsx"
+    code={dedent`
+        // employees/EmployeeForm.tsx
+        const createEmployeeMutation = gql\`
+            mutation CreateEmployee($input: CreateEmployeeInput!) {
+                createEmployee(input: $input) {
+                    id
+                    name
+                }
+            }
+        \`;
+        // ...
+        <FinalForm
+            mode="add"
+            onSubmit={(values) => {
+                // ...
+            }}
+        >
+            <Field label="Name" name="name" component={FinalFormInput} />
+            <SaveButton type="submit" />
+        </FinalForm>
+`}
+/>
+
+#### `apolloClient.mutate()` vs `useMutation()` Hook
+
+Prefer `apolloClient.mutate()` over the `useMutation()` hook to prevent the introduction of unnecessary new variables.
+Thereby, you avoid a difficult variable naming process, and your code stays short and precise.
+
+**✅ Do this:**
+
+<Source
+    language="tsx"
+    code={dedent`
+        const apolloClient = useApolloClient();
+        // ...
+        <FinalForm
+            mode="add"
+            onSubmit={async (values) => {
+                await apolloClient.mutate({
+                    mutation: createEmployeeMutation,
+                    variables: {
+                        input: {
+                            name: values.name
+                        }
+                    }
+                })
+            }}
+        >
+            <Field label="Name" name="name" component={FinalFormInput} />
+            <SaveButton type="submit" />
+        </FinalForm>
+`}
+/>
+
+**❌ Avoid this:**
+
+<Source
+    language="tsx"
+    code={dedent`
+        // you need to bother with naming this variable
+        const const [createEmployee] = useMutation(createEmployeeMutation);
+        // ...
+        <FinalForm
+            mode="add"
+            onSubmit={async (values) => {
+                // you don't see at first glance, that this is a call to a GraphQL API
+                await createEmployee({
+                    variables: {
+                        input: {
+                            name: values.name
+                        }
+                    }
+                })
+            }}
+        >
+            <Field label="Name" name="name" component={FinalFormInput} />
+            <SaveButton type="submit" />
+        </FinalForm>
+`}
+/>
+
+#### Mutation Variables
+
+A mutation should have an `input` variable that bundles all changeable attributes.
```

Comment:
This is more an API design recommendation, do we need this in Comet Admin docs?

Reply:
You are right, it's not really admin-related